app/javascript/components/GeneralInfrastructureMapping/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Field, reduxForm } from 'redux-form';
+import { renderField } from '../../common/formUtilities/renderField';
+import { required } from '../../common/formUtilities/validation';
+
+const GeneralInfrastructureMappingContainer = props => (
+  <form className="form-horizontal">
+    <Field
+      name="name"
+      label={__('Name')}
+      component={renderField}
+      validate={[required]}
```

Comment:
I think we need to pass a required flag, so we can draw the asterisk next to required fields?

Reply:
- In Classic UI we do not mark the Required fields with asterisk, but instead use a red-border for the Required field and also specify `Required` underneath the field (as shown in the above screenshot)
- In Foreman, we mark the Required fields with asterisk, and no red borders and no text underneath

So should we combine both styles here?
An asterisk + a red border box with `Required` text underneath

Reply:
Tried using the npm package - `redux-form-validators` (https://github.com/priley86/miq_v2v_ui_plugin/pull/42#discussion_r167739751), and noticed that it adds it's own Help text - `is required`, as shown below.
![image](https://user-images.githubusercontent.com/1538216/36172852-062140fa-10bc-11e8-9552-691155cf2938.png)

(Probably not a good idea to use `redux-form-validators`  for `Required` fields)